### PR TITLE
Get surfly settings from user

### PR DESCRIPTION
**Pull request by mirhmousavi**
Fixes surfly/it#3578

#### Description

Change:
- js prettier was run on the code before making changes
- add a test html page with Surfly credentials
- use new option names

[The current version](https://github.com/surfly/surfly-nic-script/files/10978812/js.script.txt)

[PR Demo](https://mirhmousavi.github.io/) 

---

#### Before requesting the review, make sure you've:
- updated all related docs
- written tests for any new functionality
- incremented `ADVISE_UPDATE` in the `surfly` tooling if an update is necessary to run this branch
- made all new strings translatable
- verified commit history
- pull request is deployed and available for testing

#### The pr is ready!
- [x] written a changelog entry
- [x] all tests are passing
- [x] I tested my changes and it works!

### CODE REVIEW

**Review by jsnjack — APPROVED** [review]
Looks okay, but how does it fix the original issue? :thinking:

**Comment by mirhmousavi** [comment]
> Looks okay, but how does it fix the original issue? thinking

Thank you for the comment. I missed a part of the ticket. I thought it was about disabling video chat inside session. I introduced two variables to control the presence of the live chat and video chat buttons. in addition to that, the user can pass their settings when initializing script via the `surflySettings` variable

**Review by jsnjack — APPROVED** [review]

File: test.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<html>
+    <head></head>
+    <body>
+        <script>
+            var showLiveChatButton  = true;
+            var showVideoChatButton = true;
+            var scaleDroneChannelId = 'fygLrCqVZQCdZfs6';
+            var nicBusNumber        = '4599130';
+            var nicChatPOC          = '1605d414-489c-4df4-83b1-334dbeb0ab2c';
```
Comment (jsnjack) [inline test.html]: This is a public repository. Looks like too many keys have been added in the commit :thinking: I think we will need to purge them from the repo history!

Comment (mirhmousavi) [reply test.html]: Thank you. I was under the impression that these are not sensitive, but removing them makes sense.
 FYI I've used `bfg` to remove the file from the git history
FYI the credentials in `main.js` doc section are not real!

**Review by aogz — APPROVED** [review]

**Pull request merged**